rep/processors/CtGovPdfProcessor.py

```diff
@@ -33,26 +33,28 @@ def _process_blob_wrapper(self, blob, source_url):
         num_list = re.findall(self.VOTE_FOR_PATTERN, page_content)
         year = source_url.split('/')[3]
 
+        # Length of the vote list and/or num_list will be 0 (or 1) if the PDF isn't a vote file that we know
+        # how to read
+        if len(votes) <= 0 or len(num_list) <= 0:
```

Comment:
> Length of the vote list and/or num_list will be 0 (or 1) if the PDF isn't a vote file that we know

is this comment saying that a length of 1 means that we _do_ support the pdf file format? Because if it's 1, this block isn't entered

Reply:
I'm not sure how that happened, I changed the comment but not the code. It should be len(votes) <= 1